client/lib/plans/features-list.js

```diff
@@ -1204,11 +1208,14 @@ export const FEATURES_LIST = {
 		getSlug: () => constants.FEATURE_PRODUCT_SCAN_DAILY_V2,
 		getIcon: () => ( { icon: 'security', component: MaterialIcon } ),
 		getTitle: () =>
+			( {
+				i5: i18n.translate( 'Scan Daily (automated)' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **7** times:
`translate( 'Daily, automated malware scanning' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/improve/pricing-page-i5-update-copy)